common/spec/specfiles.go

```diff
@@ -41,6 +41,7 @@ func CreateSpecFromFile(specFilePath string, specVars map[string]string) (spec *
 
 type File struct {
 	Aql                     utils.Aql
+	PathMapping             utils.PathMapping `json:"mappings,omitempty"`
```

Comment:
Can we leave it "pathMapping"? Or use "mapping"?
```suggestion
	PathMapping             utils.PathMapping
```

Reply:
I tried to keep the convention, I saw in lifecycle directory the use of pattern/target (which later on transform to "mappings") as "PathMappings", thus tried to keep it the same. I can change it to `Mappings` in the `File struct` but eventually its similar with lifecycles "PathMapping" which I preferred not to touch / delete / modify.
WDYT?
youll see what im talking about by searching in the code (jfrog-client-go):
```
 type Modifications struct {
	PathMappings []utils.PathMapping `json:"mappings"`
}

type PathMapping struct {
	Pattern string
	Target  string
}
```

I do think that changing to "mappings" will be better because thats how distribution server expects it eventually